assignments/lambda-classes.js

```diff
@@ -1 +1,183 @@
 // CODE here for your Lambda Classes
+class Person {
+    constructor(attr) {
+        this.name = attr.name;
+        this.age = attr.age;
+        this.location = attr.location;
+        this.gender = attr.gender;
+    }
+    speak() {
+        const phrase = `Hello my name is ${this.name}, I am from ${this.location}.`
+        console.log(phrase)
+        return phrase;
+    }
+}
+
+class Instructor extends Person {
+    constructor(instAttr) {
+        super(instAttr);
+        this.specialty = instAttr.specialty;
+        this.favLanguage = instAttr.favLanguage;
+        this.catchPhrase = instAttr.catchPhrase;
+    }
+    demo(subject) {
+        const demoPhrase = `Today we are learning about ${subject}.`;
+        console.log(demoPhrase);
+        return demoPhrase;
+    }
+    grade(Student, subject) {
+        const gradePhrase = `${Student.name} receives a perfect score on ${subject}`;
+        console.log(gradePhrase);
+        return gradePhrase;
+    }
+
+    //Stetch Task
+    gradingSystem(Student) {
+        if (Math.random() > .5) {
+            Student.grade - 3;
+        } else {
+            Student.grade + 3;
+        }
+        return Student.grade;
+    }
```

Comment:
If you are trying to change a variable's value, you have to use the assignment operator (=). So this would work better if you did something like:
```
if (Math.random() > .5) {
            Student.grade -= 3;
        } else {
            Student.grade += 3;
        }
        return Student.grade;
```
or
```
if (Math.random() > .5) {
            Student.grade = Student.grade - 3;
        } else {
            Student.grade = Student.grade + 3;
        }
        return Student.grade;
```